pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
 ]
 dependencies = [
     "apache-airflow>=2.0",
-    "apache-airflow-providers-http",
+    "apache-airflow-providers-http>=2.0.0",
```

Comment:
Any reason why we are upgrading this?

Reply:
As I understand from my investigation versions less then 2 does not make sense for this

Reply:
@a-chumagin After giving it some more thought, I believe that CI should not fail even without this upgrade. Simply bumping the version without corresponding code cleanup seems like an incomplete change.

We’ve opened an issue https://github.com/astronomer/dag-factory/issues/387 to address the version bump of the HTTP provider.

Additionally, we observed some inconsistencies with the Airflow version in this project. We are running tests in CI for Airflow >= 2.2, while the dependency requires Airflow >= 2.0. To resolve this, we’ve created an issue https://github.com/astronomer/dag-factory/issues/385.

I kindly request that you revert this upgrade in this PR and submit a new one specifically for the version bump.

Reply:
@pankajastro I agree. PR should reflect only one problem :) I reverted version restriction . Let's wait tests